SPECS/perl-Net-SSLeay/perl-Net-SSLeay.spec

```diff
@@ -19,7 +19,7 @@ BuildRequires:  perl(English)
 BuildRequires:  perl(ExtUtils::MakeMaker)
 BuildRequires:  perl-generators
 %if 0%{?with_check}
-BuildRequires:  perl(Autoloader)
+BuildRequires:  perl(AutoLoader)
```

Comment:
@dmcilvaney, do you recall what was exactly the failure, which helped us find this TDNF behaviour? I did a quick check running `tdnf install perl(Autoloader)` and `tdnf provides perl(Autoloader)` and both worked fine. DNF wouldn't work, so I think we still should be strict about casing, but I want to understand the failure better in case there's something more lingering there.

Reply:
Was this a M2 or AZL3 version of `tdnf`?

Reply:
Good question. As far as I could tell, the behaviour is the same on both. Although 3.0 TDNF is faster.:)